.gitlab-ci.yml

```diff
@@ -183,9 +183,7 @@ deploy-to-dev:
     - az containerapp revision copy --name portal --resource-group rg-ecc-portal-uks-dev --yaml revision.yml --subscription "Essex County Council (Portal)"
   rules:
     - if: '$CI_COMMIT_BRANCH == "develop"'
-      when: manual
     - if: '$CI_COMMIT_BRANCH =~ /^release\//'
-      when: manual
```

Comment:
Can we keep this line so only develop branch is deployed automatically to Dev? I recently changed deploy-to-preprod so releases can be deployed (manually) to Preprod without going to Dev first.